pkg/providers/tinkerbell/reconciler/reconciler.go

```diff
@@ -33,15 +40,17 @@ type IPValidator interface {
 
 // Scope object for Tinkerbell reconciler.
 type Scope struct {
-	ClusterSpec      *c.Spec
-	isRollingUpgrade bool
+	ClusterSpec   *c.Spec
+	operationType string
```

Comment:
if you want to use this as something like an enum, I would suggest a new type like `type clusterOperation string`